packages/crw-wallet/src/crypto.rs

```diff
@@ -141,7 +158,8 @@ impl MnemonicWallet {
 
     /// Gets the public key derived from the mnemonic.
     pub fn get_pub_key(&self) -> PublicKey {
-        self.keychain.ext_public_key.public_key
+        let compressed_public_key = self.keychain.ext_public_key.public_key.serialize();
+        PublicKey::from_slice(&compressed_public_key).unwrap()
```

Comment:
Is it safe to `unwrap()` here?

Reply:
Yes is just a conversion from the `secp256k1::PublicKey` to the `bitcoin::PublicKey` struct. The serilized public key is just was is expecting the `from_slice` method to create a new `bitcoin::PublicKey`